src/Admin/AbstractAdmin.php

```diff
@@ -1351,22 +1348,12 @@ public function getSideMenu($action, AdminInterface $childAdmin = null)
         return $this->menu;
     }
 
-    /**
-     * Returns the root code.
-     *
-     * @return string the root code
-     */
-    public function getRootCode()
+    public function getRootCode(): string
     {
         return $this->getRoot()->getCode();
     }
 
-    /**
-     * Returns the master admin.
```

Comment:
I think this comment should stay, it's helpful

Reply:
Same here

Reply:
Ah my bad, it's in the interface already.